frontend-implicaction/src/app/user/components/experience-detail/experience-detail.component.html

```diff
@@ -26,18 +26,22 @@ <h4 *ngIf="!isEditing; else experienceLabelEdition" class="pt-1">{{experience.la
       </div>
     </ng-template>
   </div>
+  <div *ngIf="isEditing" class="col-1 align-self-start text-end">
+    <i-feather (click)="deleteExperience(experience)" class="cursor-pointer text-danger" name="trash-2"></i-feather>
+  </div>
 </div>
 <div class="mb-2">
   <p *ngIf="!isEditing; else experienceDescriptionEdition" class="text-secondary">{{experience.description}}</p>
   <ng-template #experienceDescriptionEdition>
-    <div class="form-floating mb-4">
+    <div class="col-11 mb-4">
+      <label [for]="guid" class="form-label px-4">Description</label>
```

Comment:
du coup on oublie cette histoire de guid. Tu peux supprimer et la classe utils et l'attribut guid et [for] et [id]